2.0/README.md

```diff
@@ -62,14 +62,15 @@
-| [Normal Tangent Mirror Test](NormalTangentMirrorTest) | ![](NormalTangentMirrorTest/screenshot/screenshot.png) | Tests an engine's ability to load supplied tangent vectors for a normal map. |
-| [Orientation Test](OrientationTest)           | ![](OrientationTest/screenshot/screenshot.png)              | Tests node translations and rotations. |
-| [Texture Coordinate Test](TextureCoordinateTest) | ![](TextureCoordinateTest/screenshot/screenshot.png)     | Shows how XYZ and UV positions relate to displayed geometry. |
-| [Texture Settings Test](TextureSettingsTest)  | ![](TextureSettingsTest/screenshot/screenshot.png)          | Tests single/double-sided and various texturing modes. |
-| [Vertex Color Test](VertexColorTest)          | ![](VertexColorTest/screenshot/screenshot.png)              | Tests if vertex colors are supported. |
+| Model                                                 | Screenshot                                                    | Description |
+|-------------------------------------------------------|---------------------------------------------------------------|-------------|
+| [Alpha Blend Mode Test](AlphaBlendModeTest)           | ![](AlphaBlendModeTest/screenshot/screenshot.png)             | Tests alpha modes and settings. |
+| [Boom Box With Axes](BoomBoxWithAxes)                 | ![](BoomBoxWithAxes/screenshot/screenshot.jpg)                | Shows X, Y, and Z axis default orientations. |
+| [Metal Rough Spheres](MetalRoughSpheres)              | ![](MetalRoughSpheres/screenshot/screenshot.png)              | Tests various metal and roughness values. |
+| [Normal Tangent Test](NormalTangentTest)              | ![](NormalTangentTest/screenshot/screenshot.png)              | Tests an engine's ability to automatically generate tangent vectors for a normal map. |
+| [Normal Tangent Mirror Test](NormalTangentMirrorTest) | ![](NormalTangentMirrorTest/screenshot/screenshot.png)        | Tests an engine's ability to load supplied tangent vectors for a normal map. |
+| [Orientation Test](OrientationTest)                   | ![](OrientationTest/screenshot/screenshot.png)                | Tests node translations and rotations. |
+| [Texture Coordinate Test](TextureCoordinateTest)      | ![](TextureCoordinateTest/screenshot/screenshot.png)          | Shows how XYZ and UV positions relate to displayed geometry. |
+| [Texture Settings Test](TextureSettingsTest)          | ![](TextureSettingsTest/screenshot/screenshot.png)            | Tests single/double-sided and various texturing modes. |
+| [Vertex Color Test](VertexColorTest)                  | ![](VertexColorTest/screenshot/screenshot.png)                | Tests if vertex colors are supported. |
+| [Texture Transform Test](TextureTransformTest)        | ![](TextureTransformTest/screenshot/screenshot.jpg)           | Tests if KHR_texture_transform extension is supported properly. |
```

Comment:
I wonder if we should make a separate section for "Extension Test Models"?  Or do we expect all the major implementations to eventually support all the KHR extensions?